src/main/webapp/data-visualization.js

```diff
@@ -296,3 +302,47 @@ function setStyle(isCountyQuery) {
     amChartsDiv.style.display = 'block';
   }
 }
+
+function initDataTable(isCountyQuery) {
```

Comment:
quick function description for consistency might be good but the function name makes it readable/understandable